tests/conftest.py

```diff
@@ -149,6 +158,8 @@ def provider_token(bot_info):
 def create_dp(bot):
     # Dispatcher is heavy to init (due to many threads and such) so we have a single session
     # scoped one here, but before each test, reset it (dp fixture below)
+    # class DictDispatcher(Dispatcher):
+    #     pass
```

Comment:
please remove

Reply:
I left that cause 
> subclassed Dispatcher in tests so we can add custom attributes, but a test failed `test_dispatcher.py/test_multiple_async_decorator` - didn't raise the error, so dispatcher has `__dict__` (If I can get some help on that test, then Dispatcher doesn't need a `__dict__`)

Reply:
but these two lines are commented now, so they don't have any effect?

Reply:
Yea I'm hoping to get rid of `__dict__` in dispatcher soon, so after I can just uncomment that code